api/marc.py

```diff
@@ -66,12 +66,16 @@ def add_web_client_urls(self, record, library, identifier, integration=None):
         qualified_identifier = urllib.quote(identifier.type + "/" + identifier.identifier, safe='')
 
         for web_client_base_url in settings:
-            url = "{}/book/{}/{}/works/{}".format(
-                web_client_base_url,
+            link = "{}/{}/works/{}".format(
                 self.base_url,
                 library.short_name,
                 qualified_identifier,
             )
+            encoded_link = urllib.quote(link, safe='')
```

Comment:
@kristojorg The work URL is built here. The first encoding is of the `qualified_identifier`. The second encoding is for the `encoded_link`. Only the `qualified identifier` is double encoded.

Reply:
Okay yep this makes sense then, thanks!